### ISSUE TITLE
Android build fails - Android.Support.V4.View.Accessibility.AccessibilityManagerCompat

### ISSUE THREAD

**Issue by Mikilll94** [Microsoft/TailwindTraders-Mobile#57]
When building (VS 2019 16.1.5) I got an error:

`
Failed to create JavaTypeInfo for class:
Android.Support.V4.View.Accessibility.AccessibilityManagerCompat/IAccessibilityStateChangeListenerImplementor due to System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\{user}\Desktop\TailwindTraders-Mobile-master\Source\TailwindTraders.Mobile\TailwindTraders.Mobile.Android\obj\Debug\90\android\src\mono\android\support\v4\view\accessibility\AccessibilityManagerCompat_AccessibilityStateChangeListenerImplementor.java'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at Xamarin.Android.Tools.Files.CopyIfStreamChanged(Stream stream, String destination)
   at Xamarin.Android.Tasks.Generator.CreateJavaSources(TaskLoggingHelper log, IEnumerable 1 javaTypes, String outputPath, String applicationJavaClass, Boolean useSharedRuntime, Boolean generateOnCreateOverrides, Boolean hasExportReference)	TailwindTraders.Mobile.Android
`

**Comment by MarcosCobena** [comment]
Hi @Mikilll94,

It seems this one —so no related to Tailwind Traders I think:

https://developercommunity.visualstudio.com/content/problem/521034/failed-to-create-javatypeinfo.html

Just there they propose a solution, it may work for you too, could you please give it a try? Let me please close this issue in the meantime.

Thanks,

—Marcos
